Aspire Budgeting/Views/SignInView.swift

```diff
@@ -2,75 +2,58 @@
 //  Aspire Budgeting
 //
-//  Created by TeraMo Labs on 10/20/19.
-//  Copyright © 2019 TeraMo Labs. All rights reserved.
-//
 
 import GoogleSignIn
 import SwiftUI
 
 struct SignInView: View {
   @EnvironmentObject var userManager: UserManager<GIDGoogleUser>
+  @Environment (\.colorScheme) var colorScheme: ColorScheme
 
   private let rootVC = UIApplication.shared.windows.first { $0.isKeyWindow }?.rootViewController
 
   var body: some View {
     VStack {
-      Group {
-        Image("logo")
-          .resizable()
-          .aspectRatio(contentMode: .fit)
-          .frame(minHeight: 20)
-        Text("Aspire")
-          .font(.custom("Nunito-Regular", size: 30))
-          .foregroundColor(.white)
-          .padding(-20)
-        Divider().background(Color.white).padding(.horizontal, 20)
-      }
-      Group {
-        Text("Link your Google Account")
-          .font(.body)
-          .foregroundColor(.white)
-        // swiftlint:disable line_length
-        Text("Aspire Budgeting requires you to either link your existing budget sheet or create a new budget sheet in your Google Drive")
-          // swiftlint:enable line_length
-          .font(.caption)
-          .foregroundColor(.white)
-          .padding(10)
-          .multilineTextAlignment(.center)
-      }
+        Text("Aspire Budgeting")
+          .font(.nunitoSemiBold(size: 20))
+          .foregroundColor(.primaryTextColor)
 
-      YoutubePlayerView().padding(.horizontal).frame(minHeight: 80).shadow(radius: 5)
+      Image.circularLogo
+        .padding(.top)
 
-      AspireButton(title: "Connect your Google account", type: .green, imageName: "google") {
-        self.userManager.signInWithGoogle(in: self.rootVC)
-      }.frame(height: 50)
+      Text("Welcome to Aspire Budgeting")
+        .font(.nunitoRegular(size: 16))
+        .foregroundColor(.primaryTextColor)
         .padding()
-        .shadow(radius: 5)
-      HStack {
-        VStack {
-          Divider().padding(.horizontal, 20)
-        }
-        Text("OR")
-        VStack {
-          Divider().padding(.horizontal, 20)
-        }
-      }
 
-      AspireButton(title: "Copy Aspire sheet to your Google Drive", type: .red) {
-        let url = URL(string: "https://aspirebudget.com")
-        UIApplication.shared.open(url!)
-      }.frame(height: 50)
+      Text("Take control of your money everywhere you go")
+        .lineLimit(2)
+        .font(.nunitoRegular(size: 14))
+        .multilineTextAlignment(.center)
+        .foregroundColor(.secondary)
+        .frame(width: 173)
+
+      Image.diamondSeparator
         .padding()
-        .shadow(radius: 5)
 
-      Spacer()
-    }.background(BackgroundSplitColorView().edgesIgnoringSafeArea(.all))
+//      if colorScheme == .light || colorScheme == .dark{
+        GoogleSignInButton(colorScheme: colorScheme)
+          .frame(height: 50)
+          .padding()
+//      }
+      
+//      Text(colorScheme == .dark ? "In dark mode" : "In light mode")
+          
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/080f65df2311b45c8d4b592dff399d3b1bb01b5f/Aspire Budgeting/Views/SignInView.swift#L46"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>SignInView.swift:46</code></td>
    </tr>
  </tbody>
</table>